### PR TITLE
Build docker image

### LINKED ISSUE #12: Publish Docker Image
Then I can switch to godolint instead of hadolint!

### PR DESCRIPTION
**Pull request by bclermont**
Fix #12, depends on #13

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,3 +6,6 @@ fmt:
 
 test:
 	@go test -cover -v ./...
+
+docker:
+	docker build --rm -t zabio3/godolint .
```
Comment (bclermont) [inline Makefile:11]: I assume https://hub.docker.com/u/zabio3 is you

Comment (zabio3) [reply Makefile:11]: Yes, this is my account so I would like to push it after merge.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.15.2-alpine3.12 as builder
+COPY . /src
+WORKDIR /src/cmd
+RUN apk add --no-cache git ca-certificates && \
+  go build -v .
+
+FROM alpine:3.12.0
+COPY --from=builder /src/cmd/cmd /bin/godolint
```
Comment (zabio3) [inline Dockerfile:8]: There is no `/src/cmd/cmd`, so isn't it `/src/cmd` ?

Comment (bclermont) [reply Dockerfile:8]: `/src/cmd` is the directory, `/src/cmd/cmd` is the output of `go build`

Comment (zabio3) [reply Dockerfile:8]: This was a misunderstanding. As you pointed out 👍

**Pull request merged**